docs/platform/linode-gpu/getting-started-with-gpu/index.md

```diff
@@ -43,9 +45,13 @@ Depending on your distribution, you may need to install lspci manually first. On
 
 ## Install Dependencies
 
-Some dependencies may need to be installed prior to installing your driver. Listed below are the commands for installing these packages on many popular distributions:
+Some dependencies may need to be installed prior to installing your driver. Listed below are the commands for installing these packages on many popular distributions.
 
-### Ubuntu 18.10
+{{< note >}}
+After running these commands, reboot your Linode from the [Cloud Manager](https://cloud.linode.com). The commands install missing kernel headers needed by NVIDIA's driver, and rebooting will ensure that they will be available.
```

Comment:
@rsyracuse does this wording sound ok? it's a revision of the previous phrasing:

`After running these commands, reboot your Linode from the [Cloud Manager](https://cloud.linode.com) to make sure you can use the missing kernel headers.`

Reply:
looks perfect